nemo/export/trt_llm/tensor_utils.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utils for tensor conversions between tensorrt, torch and numpy."""
+
+from typing import Dict
+
+import numpy as np
+import tensorrt as trt
```

Comment:
Please try not to do unguarded trt import.

Reply:
Himm, any reason?

This is again an internal function and will be only used if the export to trt-llm is called.

Reply:
Ok, added try except to the nemo.export.__init__.py file so that trt and trt-llm imports won't raise any error if they do not exist. Do this work?